src/azurestorage.cpp

```diff
@@ -202,18 +202,13 @@ StatusCode AzureStorageBlob::fileModificationTime(int64_t* mtime_ns) {
 
         auto nanoseconds = time.to_interval();
 
-        SPDLOG_TRACE("AS: modification time for {} is {}", fullPath_, nanoseconds);
+        SPDLOG_LOGGER_TRACE(azurestorage_logger, "AS: modification time for {} is {}", fullPath_, nanoseconds);
```

Comment:
I think since we are using azurestorage_logger we can remove "AS:" from begining. Can be done in next MR.